### DISCUSSION TITLE
what is purpose of code_execution_config

### DISCUSSION THREAD

**Discussion by skeletonwarrior** [microsoft/autogen#312] [Q&A]
What is the purpose of code_execution_config, 
especially what is work_dir and what values ​​can it use?

**Comment by taylor-ennen** [comment]
Definitively this dictionary is related directly to the environment in which the agent can run code from the initialized conversation. 

As far "what" values can be called a good place to start looking is in this relative path: 
`\autogen\agentchat\user_proxy_agent.py`
Within that docstring you'll find the the values in question.
I hope that is helpful and gets you started 👍🏻
